apps/jubmoji-quest/src/pages/_document.tsx

```diff
@@ -1,13 +1,17 @@
-import { Html, Head, Main, NextScript } from 'next/document'
+import { Html, Head, Main, NextScript } from "next/document";
 
 export default function Document() {
   return (
     <Html lang="en">
       <Head />
+      <meta
```

Comment:
this will prevent the zoom that occurs on iPhone when you select a field, In our case the search input will zoom the app, and in pretty annoying to go back to the previous default one. 

`With zoom on focus`

![Simulator Screenshot - iPhone 15 - 2023-10-25 at 10 57 03](https://github.com/jubmoji/jubmoji.quest/assets/20865711/30653465-2017-47e0-b5ff-2926abd4373d)